roles/web/templates/openorchestra.vcl.j2

```diff
@@ -13,30 +13,24 @@ acl invalidators {
   "127.0.0.1";
 }
 
-backend f1 {
-  .host = "127.0.1.1";
-  .port = "{{ apache_main_port }}";
-}
-
-backend f2 {
+{% for apache_main_port in apache_main_ports %}
+backend f{{ loop.index }} {
   .host = "127.0.1.1";
   .port = "{{ apache_main_port }}";
 }
+{% endfor %}
 
 director front round-robin {
+{% for apache_main_port in apache_main_ports %}
     {
-        .backend = f1;
-    }
-    {
-        .backend = f2;
+        .backend = f{{ loop.index }};
     }
+{% endfor %}
 }
 
 sub vcl_recv {
     set req.backend = front;
 
-    unset req.http.cookie;
```

Comment:
I don't think so, by letting this line, it is no more possible de log in the BO with varnish